libs/designsystem/src/lib/components/calendar/calendar.component.scss

```diff
@@ -2,11 +2,13 @@
 $month-navigator-width: 80px;
 
 table.calendar-table {
-  width: 100%;
+  width: calc(100% - 2 * #{size('xxxs')});
   border-collapse: collapse;
   user-select: none;
   outline: none !important;
   border-spacing: 0;
+  margin: size('xxxs');
+  margin-top: 0;
```

Comment:
Did you consider to separate container and content styles? You can use a container element for the table to take care of the margins. Then you could leave the table width at 100% and let `.calendar-table` be table-related styles only

Reply:
This is a good point, but I just noticed that the vertical border below the weekdays should go all the way to the edge, so I will have to change this anyways. 

I can't really see any alternatives to styling the individual parts of the calendar with left and right spacing. Can you?